autoload/multiple_cursors.vim

```diff
@@ -1207,8 +1207,16 @@ function! s:wait_for_user_input(mode)
     call feedkeys(s:char)
     call s:cm.reset(1, 1)
     return
-  elseif s:from_mode ==# 'n'
+  elseif s:from_mode ==# 'n' || s:from_mode =~# 'v\|V'
```

Comment:
I don't think it covers all modes except `i`.  It just covers `i, v and V`. I think we should not add this behaviour to command mode and replace mode, besides insert mode. (from the top of my head)